assets/css/styleguide.scss

```diff
@@ -858,3 +883,126 @@ code[class*="language-"], pre[class*="language-"] {
 .usa-code-sample pre[class*="language-"] {
   margin-top: 0;
 }
+
+.preview {
+  .usa-background-dark {
+    display: inline-block;
+    padding: {
+      left: 1em;
+      right: 1em;
+    }
+  }
+}
+
+.usa-link-visited {
```

Comment:
@msecret are you saying we should add this class along with `a:visited` in typography.scss?